Dwarf.cpp

```diff
@@ -1,7 +1,26 @@
 #include "Dwarf.h"
+#include "Utility.h"
 
-//Dwarf::Dwarf
+Dwarf::Dwarf(std::string nameGiven, int hitPoints, int armor ) : Character(hitPoints, armor, 4), name(nameGiven)
+{
+    helpfulItems = makeHelpfulItems(2);
+    defensiveItems = makeDefensiveItems(5);
+}
+Dwarf::~Dwarf()
+{
+    
+}
+const std::string& Dwarf::getName()
+{
+    return name;
+}
+std::string Dwarf::getStats()
+{
+    return getCharacterStats(this);
+}
+void Dwarf::attack(Character& other)
+{
+    std::cout << name << " is attacking " << other.getName() << "!!!" << std::endl;
+    Character::attack(other);
+}
```

Comment:
This doesn't do anything that the base class function doesn't already do, except add some extra exclamation points.
- [ ] remove it.